lib/sensitive_data_filter/middleware/parameter_parser.rb

```diff
@@ -27,10 +27,14 @@ def can_parse?(content_type)
 
       def parse(params)
         @parse.call params
+      rescue
```

Comment:
Personally I would have the parser handle it's possible exceptions within its `call` method and then return the untouched params. If someone then chooses to extend the gem with another parser that doesn't fail gracefully they suffer the consequences.